packages/common/src/models/index.ts

```diff
@@ -287,5 +287,8 @@ export async function fetchModelResources(
  * @return {*} boolean
  */
 function shouldClearEmptyFields(item: IItem) {
-  return ["description", "snippet"].some((field) => item[field] === "");
+  return ["description", "snippet", "tags", "categories"].some(
+    (field) =>
+      item[field] === "" || (item[field]?.length === 1 && item[field][0] === "")
```

Comment:
hmm, this seems really specific. Why are we getting an array back with an empty string? Is that a result of the combobox field implementation? If so, it seems to me this logic should be handled by the field e.g. we should emit an empty array if nothing has been selected

Reply:
Good point, I refactored the logic in combobox to return an empty array